src/community/oseo/oseo-stac/src/main/java/org/geoserver/ogcapi/stac/functions/ServiceLinkFunction.java

```diff
@@ -33,6 +33,7 @@ public ServiceLinkFunction() {
         this.functionName = NAME;
     }
 
+    @Override
```

Comment:
These should not have failed the build, QA is not active on community modules (besides formatting checks). But does not hurt to add, for sure, less work down the road, if/when the module graduates.

Reply:
No, not this one(s). But since it was an automated eclipse code clean-up run.... shouldn't hurt. The offending ones on the `release` profile where
```
:100644 100644 4d4fb773ac c6cf53d7fd M  src/extension/dxf/core/src/main/java/org/geoserver/wfs/response/DXFOutputFormat.java
:100644 100644 3a65576b2a 24e9684436 M  src/wfs/src/main/java/org/geoserver/wfs/json/GeoJSONGetFeatureResponse.java
```
Those have methods with missing override annotations merged to main after the QA build for #4822, which is odd. Maybe we should run at least the QA build on `main` when a pr is merged?

Reply:
I looked at GeoJSONGetFeatureResponse
it has https://github.com/geoserver/geoserver/commit/d2a619f760aabade2cd042b59a6c1736f88196a3#diff-19b4c124f68280f5d66ce11ae42efb1e48ff8f8c57c85432e70a7647f0d3f6a4

which is an older commit than the one bringing in the PMD override check, but the earlier commit was merged later.

Because the branch it was in was out-of-date wrt the main branch; there is a checkbox in the repo settings to enforce that a branch should be up-to-date (requiring authors to update their branch after someone else's is merged - people will hate that) 
Normally this kind of thing is only temporary and should resolve itself